test-result-summary-client/src/Dashboard/Widgets/BuildStatus/BuildStatus.jsx

```diff
@@ -142,6 +142,22 @@ export default class BuildStatus extends Component {
                 const build = builds[listItem];
                 if (!build || !build.url) return null;
 
+                //Set build platform value
+                let platformRegexOpenJ9 = /-((win|linux|aix|osx|zos)\_(.*))/i;
+                let platformRegexAdopt = /_((x86|x32|aarch|ppc|arm|s390x)(.*))/i;

```

Comment:
I am wondering if we can do this in a more robust way. Could we split based on `-` or `_`? Matching with the platform may cause issues when a new platform is added in the future.